optimus_manager/xorg.py

```diff
@@ -70,7 +70,7 @@ def is_there_a_MHWD_file():
     return os.path.isfile("/etc/X11/xorg.conf.d/90-mhwd.conf")
 
 
-def do_xsetup(requested_mode):
+def do_xsetup(requested_mode, config, bus_ids):
```

Comment:
Nitpick, but could you have the argument order be `config, bus_ids, requested_mode` instead ? It's just a convention I follow to put constant values first.